index.html

```diff
@@ -0,0 +1,1040 @@
+                                Telecommunications
+                            </a>
+                            <a href="#" class="lf-body-links">
+                                <i class="fas fa-briefcase"></i>
+                                Expedition Growth Capital
+                            </a>
+                        </div>
+                        <div class="lf-footer">
+                            <a href="#">Ringover raises $12 million to bring enterprise communications to the cloud
+                                <span>on VentureBeat.</span></a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <!-- Categories -->
+        <div class="categories-container">
+            <div class="categories-content">
+                <div class="categories-title">
+                    <h3>Hard Fork <i class="fas fa-chevron-right"></i></h3>
+                </div>
+                <div class="categories-img ct-img-1">
+                    <div class="categories-img-txt">
+                        <h4>MOST POPULAR</h4>
+                        <a href="#" class="ct-img-txt-main">
+                            GameStop stocks hit record high — thanks to Redditors
+                        </a>
+                        <p class="ct-img-txt-sub">
+                            <a href="#">Ivan Mehta</a> - 1 day ago
+                        </p>
+                    </div>
+                </div>
+                <div class="categories-subtitle-img">
+                    <div class="subtitle-left">
+                        <p class="subtitle-tile">LATEST</p>
+                        <p class="subtitle-txt">
+                            Alibaba shares jump 8% as Jack Ma surfaces after 3 months
+                        </p>
+                    </div>
+                    <div class="subtitle-right ct-sub1">
+                    </div>
+                </div>
+                <div class="categories-links">
+                    <ul class="categories-ul">
+                        <li class="categories-li">
+                            Twitter shares fall 7% in pre-market trading after Trump's removal
+                        </li>
+                        <li class="categories-li">
+                            Winklevoss comes out of meditation to witness Bitcoin's record high
+                        </li>
+                        <li class="categories-li">
+                            CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%
+                        </li>
+                        <li class="categories-li">
+                            CD Projekt stock down 20% since Cyberpunk reviews hit the internet
+                        </li>
+                        <li class="categories-li">
+                            Airbnb to be bigger than eBay after its long-awaited IPO
+                        </li>
```

Comment:
- [ ] As was pointed out to you before, these `li`tags should be wrapped in an `a` tag as it should be a clickable link.